hiro/utils.py

```diff
@@ -20,12 +20,15 @@ def timedelta_to_seconds(delta):
 def time_in_seconds(value):
     """
     normalized either a datetime.date, datetime.datetime or float
-    to a float
+    to a float corresponding to a UTC timestamp.
+
+    Naive datetime objects (without timezone information) are taken in the
+    local timezone.
```

Comment:
I feel that this changes the meaning of this method without properly accounting for its users.

See https://github.com/alisaifee/hiro/blob/a8edda4a8b17236085e433afc43d7a9aa9a95883/hiro/core.py#L164-L165 for instance, where the `offset` from "now" is counted as `time_in_seconds(start) - time.time()`. The latter is a unix timestamp in UTC, while the former (as of your PR) is now skewed by the tz offset.

Reply:
No, it is not. That's exactly the misbehavior I'm fixing with this PR.

Reply:
`datetime.datetime.utcnow()` also returns a naive timestamp.

Reply:
Alright, nevermind, I see now that @alisaifee was operating with the latter assumption from the start. Let's go with that then.